addons/event/models/event_event.py

```diff
@@ -176,6 +176,20 @@ def _default_event_mail_ids(self):
         'Start sale date', compute='_compute_start_sale_date',
         help='If ticketing is used, contains the earliest starting sale date of tickets.')
 
+    def name_get(self):
+        if not self.env.context.get('registration'):
+            return super().name_get()
+        res = []
+        for event in self:
+            name = (
+                _("%s (%s seats remaining)",
```

Comment:
Wild that give "Free Beer (Unlimited seats remaining)" ? Isn't it a bit long, maybe "Free Beer (unlimited seats)" ?